src/Model/Scope/Scope.php

```diff
@@ -208,18 +208,14 @@ public function toWords(bool $asHtml = false): string
     /**
      * Create a scope from array of scopes or arrays.
      *
-     * @param mixed  $scopeOrArray
+     * @param string|array  $scopes
      * @param string $junction
      *
      * @return static
      */
-    public static function create($scopeOrArray = null, $junction = self::AND)
+    public static function create($scopes = null, $junction = self::AND)
```

Comment:
bad design - should user use `new Cl` or this?

Reply:
Both are possible and equal.

Reply:
I would only endorse static create() if it would be returning instances of various classes. Since it always returns new Scope, there is no benefit in having a static call, so I propose not to have this method. 

it would confuse by offering two identical implementations for no benefit.

Reply:
So `create` methods removed